CurrencyConverted.playground/Contents.swift

```diff
@@ -0,0 +1,44 @@
+import UIKit
+
+enum Currency {
+    case cad
+    case mxn
+}
+
+
+let usToCad = 1.41
+let usToMxn = 24.05
+
+var currency: Currency = .mxn
+
+func convert(_ dollars: Double) -> Double {
+    
+    var convertedValue: Double = 1.0
+    
+    switch currency {
+    case .cad:
+        convertedValue = dollars * usToCad
+    case .mxn:
+        convertedValue = dollars * usToMxn
+    }
+    
+    return convertedValue
+}
+
+
+
+func convert(amountString: String) -> String? {
+    
+    guard let amount = Double(amountString) else { return nil }
+    
+    let usDollar = convert(amount)
+    
+    return String(usDollar)
+}
+
+
+
+convert(amountString: "78")  //  .cad
+
+convert(amountString: "154") //  .mxn
```

Comment:
I would maybe suggest putting setting the currency before each of these calls. You have them marked .cad and .mxn in the comments, but really both of these lines are converting to mxn, as that is what is set above...

``` swift
currency = .cad
convert(amountString: "78")  //  .cad
currency = .mxn
convert(amountString: "154") //  .mxn
```